lib/matplotlib/axes/_axes.py

```diff
@@ -4296,21 +4296,10 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             See :mod:`matplotlib.markers` for more information about marker
             styles.
 
-        cmap : str or `~matplotlib.colors.Colormap`, default: :rc:`image.cmap`
-            A `.Colormap` instance or registered colormap name. *cmap* is only
-            used if *c* is an array of floats.
-
-        norm : `~matplotlib.colors.Normalize`, default: None
-            If *c* is an array of floats, *norm* is used to scale the color
-            data, *c*, in the range 0 to 1, in order to map into the colormap
-            *cmap*.
-            If *None*, use the default `.colors.Normalize`.
-
-        vmin, vmax : float, default: None
-            *vmin* and *vmax* are used in conjunction with the default norm to
-            map the color array *c* to the colormap *cmap*. If None, the
-            respective min and max of the color array is used.
-            It is an error to use *vmin*/*vmax* when *norm* is given.
+        cmap, norm, vmin, vmax
+            The data normalization and colormapping, which are explained in
+            detail in the documentation for `~matplotlib.axes.Axes.imshow`.
```

Comment:
```suggestion
            Data normalization and colormapping parameters for *c*. See
            `~matplotlib.axes.Axes.imshow` for a detailed description.
```
I'd like to have *c* prominently mentioned at the beginning, so that one knows what's affected without having first to read to the end. The rest of the rewording sort of follows from that.